protocols/twitter/twitter.c

```diff
@@ -673,6 +673,15 @@ static void twitter_handle_command(struct im_connection *ic, char *message)
 		message = new = g_strdup_printf("@%s %s", bu->handle, cmd[2]);
 		in_reply_to = id;
 		allow_post = TRUE;
+	} else if (g_strcasecmp(cmd[0], "rawreply") == 0 && cmd[1] && cmd[2]) {
+		id = twitter_message_id_from_command_arg(ic, cmd[1], NULL);
+		if (!id) {
+			twitter_log(ic, "Tweet `%s' does not exist", cmd[1]);
+			goto eof;
+		}
+		message += (cmd[2] - cmd[0]);
```

Comment:
I'm not even sure what I was thinking when I wrote that (this patch was written 6 months ago). To be honest, I'm pretty sure the current implementation relies on split_command_parts not being re-written, so I'll push through a patch to just use "message = cmd[2]".